prog/vm/nexus.rb

```diff
@@ -202,25 +202,29 @@ def before_run
       host_exclusion_filter = frame["exclude_host_ids"] || []
       gpu_count = frame["gpu_count"] || 0
       runner = GithubRunner.first(vm_id: vm.id) if vm.location_id == Location::GITHUB_RUNNERS_ID
-      allocation_state_filter, location_filter, location_preference, host_filter =
+      allocation_state_filter, location_filter, location_preference, host_filter, family_filter =
```

Comment:
unrelated to this PR, but I'm wondering if we should rename `host_filter` to something like `id_filter`.

Reply:
Makes sense. I might change it at https://github.com/ubicloud/ubicloud/pull/3281. 

It also applies to host id exclusion. May be `id_exclusion_filter`